src/utils/debug.js

```diff
@@ -43,3 +44,18 @@ export default function logError(error) {
 
   console.groupEnd();
 }
+
+export function translateErrorForUser(error) {
+  let text;
+  if (error instanceof CoreError) {
+    if (error.code == ErrorCodes.INSUFFICIENT_FUNDS) {
+      text = translate('ErrorCodes.CoreErrorInsufficientFunds');
+    }
+  } else if (error instanceof TransferError) {
+    /*eslint-disable no-empty */
```

Comment:
do we want these other cases here even if we are not doing anything?

Reply:
I was thinnking it can be usefull for another task with errors but I agree we can remove it